dotCMS/src/main/resources/dotmarketing-config.properties

```diff
@@ -902,3 +902,5 @@ api.cors.default.Access-Control-Expose-Headers=*
 
 # IGNORE_REFERER_FOR_PATHS=/html/common/css.jsp,/html/my-plugin/*
 
+#if it is tru JWT is use in Push Publish
+#USE_JWT_TOKEN_IN_PUSH_PUBLISH = false
```

Comment:
Remove this. We will only document the property but won't include it here

Reply:
done https://github.com/dotCMS/core/pull/19273/commits/303cd0bf044e9265865287ea596719a4346e2895#diff-9eef290d7e7caa46fa38e7f59dfe65cdL905